src/server.js

```diff
@@ -0,0 +1,94 @@
+console.log("Hello World!");
```

Comment:
Please, group all `require` invokes at the beginning.

Reply:
Done